Sources/Card/CardClient.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import PaymentsCore
+
+public class CardClient {
```

Comment:
General question - does this class need to be public or can it be gated to only the Card module?

Reply:
ah this needs to be public so merchants can use it :smile:

Reply:
Yeah I think for the most part `CardClient` and anything in its public interface ended up needing to be `public` for Android as well. Everything else we were able to make internal, with the exception of a few classes in our `Core` module.